File: app/src/main/java/com/hao/m/ui/main/recommend/RecommendAdapter.java
package com.hao.m.ui.main.recommend;

import com.chad.library.adapter.base.BaseQuickAdapter;
import com.chad.library.adapter.base.BaseViewHolder;
import com.hao.m.R;

import java.util.List;

public class RecommendAdapter extends BaseQuickAdapter<String, BaseViewHolder> {
    public RecommendAdapter(List data) {
        super(R.layout.item_user_show, data);
    }

    @Override
    protected void convert(BaseViewHolder helper, String item) {
        helper.setText(R.id.tvUserName, item);
    }
}

File: app/src/main/java/com/hao/m/ui/main/recommend/RecommendAdapter.kt
package com.hao.m.ui.main.recommend

import com.chad.library.adapter.base.BaseQuickAdapter
import com.chad.library.adapter.base.BaseViewHolder
import com.hao.m.R
/**
*@author : HaoBoy
*@date : 2018/9/21
*@description : 发现页adapter
**/
class RecommendAdapter(data: List<String>) : BaseQuickAdapter<String, BaseViewHolder>(R.layout.item_user_show, data) {

    override fun convert(helper: BaseViewHolder, item: String) {
        helper.setText(R.id.tvUserName, item)
    }
}
